sql/database.sql

```diff
@@ -8723,6 +8723,7 @@ CREATE TABLE `procedure_questions` (
 DROP TABLE IF EXISTS `procedure_order`;
 CREATE TABLE `procedure_order` (
   `procedure_order_id`     bigint(20)   NOT NULL AUTO_INCREMENT,
+  `uuid`                   binary(16)   DEFAULT NULL
```

Comment:
missing comma at end (this is what is breaking travis)

Reply:
```suggestion
  `uuid`                   binary(16)   DEFAULT NULL,
```

Reply:
Ooops!

```suggestion
  `uuid`                   binary(16)   DEFAULT NULL,
```